### PR TITLE
Create question

### PR DESCRIPTION
**Pull request by alappies**

### CODE REVIEW

**Review by WalkerTexas-coder — DISMISSED** [review]
alright looks good!

File: app/controllers/questions_controller.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 class QuestionsController < ApplicationController
+    def set_current_user
+        User.current = current_user
+      end
```
Comment (VonTeacher) [inline app/controllers/questions_controller.rb:4]: Small comment, but proper alignment of this methods contents and `end` and a line break after this method will help keep your code clear.

File: app/javascript/pages/QuestionNew.js [hunk 2]
```diff
@@ -1,11 +1,66 @@
...
+        this.setState({submitted: true})
+      }
     render() {
         return (
-            <div>
-                This is the Question New Index
-            </div>
+            <Box id="form"
+            component="form"
+            sx={{ '&.MuiTextFeild-root': { m: 1, width: '25ch' },}}
```
Comment (VonTeacher) [inline app/javascript/pages/QuestionNew.js:35]: Is `Feild` a typo? Should it be `Field`?

File: app/javascript/pages/ProtectedIndex.js [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
+import Button from "@mui/material/Button";
 
 export default class ProtectedIndex extends Component {
     render() {
         return (
             <div>
                 This is the Protected Index
+                <NavLink to={"/questionnew"}>
+              <Button variant="text">ADD QUESTION</Button>
+            </NavLink>
```
Comment (WalkerTexas-coder) [inline app/javascript/pages/ProtectedIndex.js:12]: Address indentation

File: app/javascript/pages/QuestionIndex.js [hunk 4]
```diff
@@ -9,6 +11,9 @@ export default class QuestionIndex extends Component {
           this.props.questions.map((question) => {
             return <QuestionCard key={question.id} question={question} />;
           })}
+          <NavLink to={"/questionnew"}>
+              <Button variant="text">ADD QUESTION</Button>
+            </NavLink>
```
Comment (WalkerTexas-coder) [inline app/javascript/pages/QuestionIndex.js:16]: Address indentation

**Review by WalkerTexas-coder — APPROVED** [review]
couple indentation fixes

File: app/javascript/pages/QuestionNew.js (hunk 2 above)
Comment (alappies) [reply app/javascript/pages/QuestionNew.js:35]: I copied and pasted from the MUI website... so this time, and only this time it wasn't my typo.

File: app/javascript/pages/ProtectedIndex.js (hunk 3 above)
Comment (alappies) [reply app/javascript/pages/ProtectedIndex.js:12]: heard

**Pull request merged**